cucumber/authenticators_oidc/features/authn_oidc_okta.feature

```diff
@@ -15,21 +19,26 @@ Feature: OIDC Authenticator V2 - Users can authenticate with Okta using OIDC
           - !variable client-id
           - !variable client-secret
           - !variable claim-mapping
-          - !variable state
-          - !variable nonce
           - !variable redirect-uri
+
           - !group users
 
           - !permit
             role: !group users
             privilege: [ read, authenticate ]
             resource: !webservice
     """
-    And I am the super-user
-    And I successfully set Okta OIDC V2 variables
+    And I add an Okta user
+    And I set conjur variables
+      | variable_id                               | value                                                         | environment_variable  |
+      | conjur/authn-oidc/okta-2/provider-uri     |                                                               | OKTA_PROVIDER_URI     |
+      | conjur/authn-oidc/okta-2/client-id        |                                                               | OKTA_CLIENT_ID        |
+      | conjur/authn-oidc/okta-2/client-secret    |                                                               | OKTA_CLIENT_SECRET    |
+      | conjur/authn-oidc/okta-2/claim-mapping    | preferred_username                                            |                       |
+      | conjur/authn-oidc/okta-2/redirect-uri     | http://localhost:3000/authn-oidc/okta/cucumber/authenticate |                       |
 
   @smoke
   Scenario: Authenticating with Conjur using Okta
-    Given I fetch a code from okta
-    When I authenticate via OIDC with code and service_id "okta-2"
-    Then The okta user has been authorized by conjur
+    Given I fetch a code from Okta
+    When I authenticate via OIDC V2 with code and service-id "okta-2"
```

Comment:
do we want to hardcode service-id of `okta-2` or parameterize for future OIDC providers like ADFS and Identity

Reply:
In this step, `okta-2` extracted and used to retrieve the specified Service ID ([step definition](https://github.com/cyberark/conjur/blob/nonce-pkce-refactor-v2/cucumber/authenticators_oidc/features/step_definitions/authn_oidc_steps.rb#L201)).

Given an OIDC V2 authenticator `adfs`, the line would be something like:
`When I authenticate via OIDC V2 with code and service-id "adfs"`
if we wanted to validate the user was authenticating with the `adfs` Service ID.